src/main/java/org/jabref/logic/remote/client/RemoteClient.java

```diff
@@ -18,7 +18,7 @@ public class RemoteClient {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(RemoteClient.class);
 
-    private static final int TIMEOUT = 200;
+    private static final int TIMEOUT = 20000;
```

Comment:
Do you have a zero to much here? Now you would have 20s...

Reply:
I do, yes! Thanks :)